util/resource/summary.go

```diff
@@ -28,9 +28,14 @@ func (ss Summaries) Duration() wfv1.ResourcesDuration {
 	// Add container states.
 	d := wfv1.ResourcesDuration{}
 	for _, s := range ss {
+		// age is converted to seconds, otherwise the multiplication below is very likely to overflow
 		age := int64(s.age().Seconds())
 		for n, q := range s.ResourceList {
-			d = d.Add(wfv1.ResourcesDuration{n: wfv1.NewResourceDuration(time.Duration(q.Value() * age / wfv1.ResourceQuantityDenominator(n).Value() * int64(time.Second)))})
+			d = d.Add(wfv1.ResourcesDuration{
+				n: wfv1.NewResourceDuration(time.Duration(
+					q.MilliValue()*age/wfv1.ResourceQuantityDenominator(n).MilliValue(),
```

Comment:
```suggestion
					q.MilliValue() * age / wfv1.ResourceQuantityDenominator(n).MilliValue(),
```
spaces would definitely make this a little easier on the eyes 😅 

Or did `gofmt` make it have no spaces?

Reply:
I agree with you, `gofmt` did that unfortunately :/